svir/dialogs/viewer_dock.py

```diff
@@ -776,34 +879,31 @@ def on_export_data_button_clicked(self):
                     '~/recovery_curves_%s.csv' %
                     self.approach_cbx.currentText()),
                 '*.csv')
-        if filename:
+        if filename is not None:
             self.write_export_file(filename)
 
     def write_export_file(self, filename):
```

Comment:
In the engine when we export CSV files, we use an extension `_XYZ.csv` where XYZ is the calculation ID. This could be a suggestion for a future PR.

Reply:
I've just opened an issue to keep track of this suggestion